addons/fleet/models/fleet_vehicle_cost.py

```diff
@@ -136,6 +136,7 @@ def default_get(self, default_fields):
     cost_amount = fields.Float(related='cost_id.amount', string='Amount', store=True)
     odometer = fields.Float(string='Odometer at creation', 
         help='Odometer measure of the vehicle at the moment of the contract creation')
+    driver_email = fields.Char(related='purchaser_id.email', store=True)
```

Comment:
Please put it right after the purchaser_id field definition to ease model understanding + be consistent in naming (pruchaser_id_email).

Reply:
Post-review: should not be necessary anymore.